### PR TITLE
Update config UI to use modern look

### PR DESCRIPTION
**Pull request by Abbie5**
emi-modern-config-look

### CODE REVIEW

File: xplat/src/main/java/dev/emi/emi/screen/widget/config/ConfigJumpButton.java [hunk 1]
```diff
@@ -26,7 +26,10 @@ public void renderWidget(DrawContext raw, int mouseX, int mouseY, float delta) {
 		if (this.isMouseOver(mouseX, mouseY)) {
 			context.setColor(0.5f, 0.6f, 1f);
 		}
-		super.renderWidget(raw, mouseX, mouseY, delta);
+		context.push();
+			context.matrices().translate(0, 0, 100);
```
Comment (emilyploszaj) [inline xplat/src/main/java/dev/emi/emi/screen/widget/config/ConfigJumpButton.java]: EMI style doesn't do indenting like this, though I admit it is reasonable

File: xplat/src/main/java/dev/emi/emi/screen/widget/config/ListWidget.java [hunk 2]
```diff
@@ -165,67 +169,36 @@ public void render(DrawContext draw, int mouseX, int mouseY, float delta) {
...
-			bufferBuilder.vertex(this.right, this.top, 0)
-				.texture((float)this.right / 32.0F, (float)(this.top + (int)this.getScrollAmount()) / 32.0F)
-				.color(32, 32, 32, 255);
-			bufferBuilder.vertex(this.left, this.top, 0)
-				.texture((float)this.left / 32.0F, (float)(this.top + (int)this.getScrollAmount()) / 32.0F)
-				.color(32, 32, 32, 255);
-			BufferRenderer.drawWithGlobalProgram(bufferBuilder.end());
+			RenderSystem.enableBlend();
+			Identifier identifier = this.client.world == null ? MENU_LIST_BACKGROUND_TEXTURE : INWORLD_MENU_LIST_BACKGROUND_TEXTURE;
+			draw.drawTexture(
```
Comment (emilyploszaj) [inline xplat/src/main/java/dev/emi/emi/screen/widget/config/ListWidget.java]: One line is fine for this instead of many.

**Review by emilyploszaj — CHANGES_REQUESTED** [review]
Looks nice! A little scary with the scissor but I'm sure it's fine. Just a few style crits.

**Review by emilyploszaj — APPROVED** [review]

**Comment by Abbie5** [comment]
I also said this in discord, but just to make sure: if possible this should not be merged to versions before 1.20.5. I made this PR to make EMI's config UI match vanilla's better on 1.20.5+, since Mojang changed it in 1.20.5 (that is the "modern look" I was referring to). It will likely look out-of-place on earlier versions, not to mention be difficult to port.

I should have mentioned this earlier, sorry.

**Pull request merged**